corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -291,26 +289,31 @@ def _normalize_sequence(self, sequence):
         return sequence
 
     def _record_datadog_metrics(self, changes_chunk, processing_time):
-        tags = {"pillow_name": self.get_name(), "mode": "chunked"}
         change_count = len(changes_chunk)
-        if settings.ENTERPRISE_MODE:
-            type_counter = Counter([
-                change.metadata.document_subtype
-                for change in changes_chunk if change.metadata.document_type == 'CommCareCase'
-            ])
-            for case_type, type_count in type_counter.items():
-                metrics_counter('commcare.change_feed.changes.count', type_count,
-                                tags={**tags, 'case_type': case_type})
-
-            remainder = change_count - sum(type_counter.values())
-            if remainder:
-                metrics_counter('commcare.change_feed.changes.count', remainder, tags=tags)
-        else:
-            metrics_counter('commcare.change_feed.changes.count', change_count, tags=tags)
+        by_data_source = defaultdict(list)
```

Comment:
this change adds the `datasource` tag which is consistent with the metrics recorded for single changes (this is for bulk changes)